lesson3/app/src/main/java/io/wolfd/mobileproto/lesson3/SettingsFragment.java

```diff
@@ -10,12 +10,15 @@
 import android.view.ViewGroup;
 
 public class SettingsFragment extends Fragment {
+    // What does Nullable do? I actually don't know.
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_settings, container, false);
 
         // basic settings here.
+        // Probably could done something sleek with a helper function to avoid 3 copy pasted blocks
+        // where all you chanced is color.
```

Comment:
Good point, I didn't really put much effort into that part of the app (because I wouldn't usually want people to be able to change the colors in my app besides a night mode thing.)